src/Stryker.Core/Stryker.Core.UnitTest/Mutators/CollectionExpressionMutatorTests.cs

```diff
@@ -0,0 +1,523 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Shouldly;
+using Stryker.Abstractions.Exceptions;
+using Stryker.Abstractions;
+using Stryker.Abstractions.Mutators;
+using Stryker.Abstractions.Options;
+using Stryker.Core.Compiling;
+using Stryker.Core.InjectedHelpers;
+using Stryker.Core.Mutants;
+using Stryker.Core.MutationTest;
+using Stryker.Core.Mutators;
+using Stryker.Core.ProjectComponents.SourceProjects;
+using System.Text;
+
+namespace Stryker.Core.UnitTest.Mutators;
+
+[TestClass]
+public class CollectionExpressionMutatorTests : TestBase
+{
+    [TestMethod]
+    public void ShouldBeMutationLevelAdvanced()
+    {
+        var target = new CollectionExpressionMutator();
+        target.MutationLevel.ShouldBe(MutationLevel.Advanced);
+    }
+
+    [TestMethod]
+    [DataRow("[]")]
+    [DataRow("[ ]")]
+    [DataRow("[           ]")]
+    [DataRow("[ /* Comment */ ]")]
+    public void ShouldAddValueToEmptyCollectionExpression(string expression)
+    {
+        var expressionSyntax = SyntaxFactory.ParseExpression(expression) as CollectionExpressionSyntax;
+        var target = new CollectionExpressionMutator();
+        var result = target.ApplyMutations(expressionSyntax, null);
+        
+        var mutation = result.ShouldHaveSingleItem();
+        mutation.DisplayName.ShouldBe("Collection expression mutation");
+        var replacement = mutation.ReplacementNode.ShouldBeOfType<CollectionExpressionSyntax>();
+        replacement.Elements.ShouldNotBeEmpty();
```

Comment:
Should we check here what the element is?

Reply:
I'll update it to do so. I think I originally didn't because I also had another mutation that added two default elements. But thinking now, they would all be check-able anyway, so it was probably just carelessness.